qatrfm/cli.py

```diff
@@ -76,9 +76,15 @@ def find_tf_file(basedir):
 @click.option('--no-clean', 'no_clean', is_flag=True,
               help="Don't clean the environment when the tests finish. "
               "This is useful for debug and troubleshooting.")
-def cli(test, path, tfvar, snapshots, no_clean):
+@click.option('--loglevel', 'loglevel', type=click.Choice([
+              'CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG']),
```

Comment:
why reinvent the wheel ? https://docs.python.org/3.1/library/logging.html#logging-levels

Reply:
it's not reinventing the wheel. These codes are passed to the python logger later, which is exactly the link you pasted. This just allows the user to select which log level to display.

Reply:
I thought something like this https://github.com/pallets/click/issues/605 , but looks like it is not supported yet ...